modules/ROOT/assets/attachments/bundled-metric.yaml

```diff
@@ -183,42 +183,72 @@ paths:
                             properties:
                               admin_net_bytes_recv:
                                 type: integer
+                                description: The total number of bytes received (since node start-up) on the network interface to which the Sync Gateway api.admin_interface is bound.
                               admin_net_bytes_sent:
                                 type: integer
+                                description: The total number of bytes sent (since node start-up) on the network interface to which the Sync Gateway api.admin_interface is bound.
                               error_count:
                                 type: integer
+                                description: The total number of errors logged.
                               go_memstats_heapalloc:
                                 type: integer
+                                description: HeapAlloc is bytes of allocated heap objects. Allocated heap objects include all reachable objects, as well as unreachable objects that the garbage collector has not yet freed. Specifically, HeapAlloc increases as heap objects are allocated and decreases as the heap is swept and unreachable objects are freed. Sweeping occurs incrementally between GC cycles, so these two processes occur simultaneously, and as a result HeapAlloc tends to change smoothly (in contrast with the sawtooth that is typical of stop-the-world garbage collectors).
                               go_memstats_heapidle:
                                 type: integer
+                                description: HeapIdle is bytes in idle (unused) spans. Idle spans have no objects in them. These spans could be (and may already have been) returned to the OS, or they can be reused for heap allocations, or they can be reused as stack memory. HeapIdle minus HeapReleased estimates the amount of memory that could be returned to the OS, but is being retained by the runtime so it can grow the heap without requesting more memory from the OS. If this difference is significantly larger than the heap size, it indicates there was a recent transient spike in live heap size.
                               go_memstats_heapinuse:
                                 type: integer
+                                description: HeapInuse is bytes in in-use spans. In-use spans have at least one object in them. These spans an only be used for other objects of roughly the same size. HeapInuse minus HeapAlloc estimates the amount of memory that has been dedicated to particular size classes, but is not currently being used. This is an upper bound on fragmentation, but in general this memory can be reused efficiently.
```

Comment:
Firstly, I want to suggest the following edit here:
* These spans an only be used -> These spans can only be used

Also, would we able to please provide any of the following details as well on `go_memstats_heapinuse`?
* When will the memory be released?
* What Sync Gateway configuration settings control/affects the behavior of this stat?

Reply:
This file is generated from API specs created by devs and brought in by us. I think this is out of the scope of this specific change and could be raised as a separate ticket for the sync gateway team. cc @osfameron who handled pulling this down.

Reply:
That sounds good @ElliotFrancisHunter I will bring this up with the team in a different forum and then maybe we can revisit. Thank you for your input.